bin/upload-app

```diff
@@ -59,9 +59,8 @@ class UploadAppEntry
     end
 
     if options[:access_key].nil?
-      $logger.warn 'An access_key is required to upload the app'
       Optimist::with_standard_exception_handling parser do
-        raise Optimist::HelpNeeded
+        raise Optimist::CommandlineError.new('An access_key is required to upload the app', 1)
```

Comment:
Looks like this is equivalent to:

```suggestion
        parser.die('An access_key is required to upload the app', 1)
```

https://github.com/ManageIQ/optimist/blob/f022eb85c24a30e195b234e5b7722711e27c36d2/lib/optimist.rb#L960-L963

https://github.com/ManageIQ/optimist/blob/f022eb85c24a30e195b234e5b7722711e27c36d2/lib/optimist.rb#L991-L993

Reply:
Changed